src/containers/App/index.tsx

```diff
@@ -43,27 +43,86 @@ class App extends React.Component<AppProps> {
             styles.container,
             { [styles.isLoading]: showLoadingOverlay },
         ]);
+
+        const { darkAlgorithm } = theme;
+
         return (
-            <div
-                className={classNames([styles.wrapper, { [styles.isLoading]: showLoadingOverlay }])}
+            <ConfigProvider
+                theme={{
+                    algorithm: darkAlgorithm,
+                    token: {
```

Comment:
optional: consider `useMemo`-ing `theme` or breaking it out into a const to avoid regenerating this object on every render